web-ng/src/app/components/observation-form/observation-form.component.ts

```diff
@@ -78,6 +78,20 @@ export class ObservationFormComponent {
     observationService
       .getSelectedObservation$()
       .subscribe((observation?: Observation | LoadingState) => {
+        if (
```

Comment:
Please define non-trivial lambdas like this one in their own function (e.g. `onSelectObservation()` or `).

Reply:
Extracted